### DISCUSSION TITLE
Problem with github education, please help!

### DISCUSSION THREAD

**Discussion by MauroGutierrez1985** [community/community#137508] [GitHub Education]
### Hi everyone,

Hello people, I’m currently facing an issue with accessing the GitHub Education benefits. As the requirement say, it is necessary to have a university email address to verify my student status. However, my college only provides such email addresses to faculty members, like teachers, not to students.
I have already raised this issue with the college administration, but unfortunately, I have not received any response or resolution.
Could you please advise on how I can proceed in this situation? Is there an alternative way to verify my student status?
Thank you !

**Comment by andrefqms** [comment]
Did you try asking github? **https://support.github.com/contact/education**

If your college doesn’t provide student email addresses, you can still try to verify your student status with GitHub Education by submitting other forms of proof, such as an enrollment verification letter or a student ID. Contact GitHub Education support directly to explain your situation and ask if they accept alternative documentation.
Documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter.

Go try open a ticket with GitHub and with a couple days later it will be fixed the issue.
I hope I have helped in some way
